config/src/config/test_data/validator.yaml

```diff
@@ -50,3 +51,7 @@ validator_network:
                 from_disk: "/full/path/to/token"
     mutual_authentication: true
     network_id: "validator"
+
+upstream:
+    networks:
+        - validator
```

Comment:
this should be empty for a validator (by default `NetworkId::Validator` is upstream)